.gitignore

```diff
@@ -0,0 +1,2 @@
+__pycache__*
```

Comment:
Might need to do some research on why this is still not catching all those files.

Reply:
I'll look into this a bit more this evening to try and figure out whats happening